src/components/map/RealtimeLayersService.js

```diff
@@ -10,14 +10,16 @@ goog.require('ga_map_service');
 
   module.provider('gaRealtimeLayersManager', function() {
     this.$get = function($rootScope, $http, $timeout, gaLayerFilters,
-        gaMapUtils, gaUrlUtils) {
+        gaMapUtils, gaUrlUtils, gaLayers) {
 
       var timers = [];
       var realTimeLayersId = [];
       var geojsonFormat = new ol.format.GeoJSON();
 
-      function setLayerSource(layer) {
-        var fullUrl = gaUrlUtils.proxifyUrl(layer.geojsonUrl);
+      function setLayerSource(layer, isNewSource) {
+        var fullUrl = isNewSource ?
+            gaLayers.getLayerProperty(layer.bodId, 'geojsonUrl') :
+            gaUrlUtils.proxifyUrl(layer.geojsonUrl);
```

Comment:
To be more simple, we have 2 solutions:

-  Add a getter to the geojsonUrl property in gaDefinePropertiesForLayer

or

-  Remove the  `geojsonUrl` property from gaDefinePropertiesForLayer and access to the property through `gaLayers.getLayerProperty` everywhere.

With one of these solution no need of `isNewSource` parameter.